modules/s2i/core/module.yaml

```diff
@@ -112,6 +112,12 @@ envs:
     Delete source files at the end of build.  Defaults to true.
   example: "false"
 
+- name: S2I_ENABLE_JLINK_WORKFLOW
```

Comment:
the variable name you've used in the scripts hasn't got the `_WORKFLOW` suffix. They need to match up. I think I prefer without `_WORKFLOW`.